python/lsst/ctrl/orca/VanillaCondorWorkflowMonitor.py

```diff
@@ -116,20 +128,23 @@ def run(self):
                     val = self._parent.handleJobOfficeEvent(jobOfficeEvent)
                 if event is not None:
                     val = self._parent.handleEvent(event)
-                    if self._parent._locked.running == False:
-                        print "and...done!"
+                    if self._parent._locked.running is False:
+                        print("and...done!")
                         return
                 elif logEvent is not None:
                     val = self._parent.handleEvent(logEvent)
-                    if self._parent._locked.running == False:
-                        print "logger handled... and... done!"
+                    if val is None:
+                        print("error receiving last log event")
                         return
-                if (jobOfficeEvent is not None) or (jobOfficeEvent is not None) or (jobOfficeEvent is not None):
+                    if self._parent._locked.running is False:
```

Comment:
that line isn't right, at all.  should be testing event and logEvent too.  Fixed.